assemblers/io.weldr.qcow2

```diff
@@ -47,6 +47,7 @@ def main(tree, input_dir, options):
     filename = options["filename"]
     partition_table_id = options["partition_table_id"]
     root_fs_uuid = options["root_fs_uuid"]
+    output_dir = options.get("output_dir", ".")
```

Comment:
This would preserve the previous behavior, which resulted in no output image on the host. Do we want to choose a different default?

Reply:
Hm, so far we made input/output mandatory. I think that makes sense, we don't want to drop files all over the place just because the user forgot to pass the option. At least for now (otherwise we should change every occurrance)